universal-login-wallet/src/ui/react/Modals/ModalWaitingFor.tsx

```diff
@@ -20,7 +25,7 @@ const ModalWaitingFor = ({action}: {action: string}) => {
         <div className="modal-pending-section">
           <ProgressBar className="modal-pending-loader"/>
           <h3 className="modal-section-title transaction-hash-title">Transaction status</h3>
-          <p className="txn-hash-text">0x5a63…f3a68a</p>
```

Comment:
- Do we need to add ... like `0x1234...FF89`?
- Can we make it a link to etherscan?

Reply:
Yes, we can. Something like this? 
```<a href=`https://etherscan.io/tx/${transactionHash}`" target="_blank">{formatTransactionHash(transactionhash)}</a> ```